documentation/modules/con-securing-client-authorization.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka-clients.adoc
+
+[id='con-securing-client-authorization-{context}']
+= User authorization
+
+User authorization is configured using the `authorization` property in `KafkaUser.spec`.
+The authorization type enabled for a user is specified using the `type` field.
+
+If no authorization is specified, the User Operator does not provision any access rights for the user.
```

Comment:
I feel we need to be explicit about whether  such users get access or not. And that depends on the authoriser. Not completely sure about this suggestion because it brings in the `AclAuthoriser` before it's really been discussed:

```suggestion
If no authorization is specified, the User Operator does not provision any access rights for the user.
Whether such a `KafkaUser` can access resources depends on the authorizer being used. For example, for the `AclAuthorizer` this is determined by its `allow.everyone.if.no.acl.found` configuration.
```

Reply:
I've moved it to the bottom of the section, so we've already introduced the options for specifying authorization.